code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -108,7 +108,7 @@
 		force = 30
 		w_class = W_CLASS_LARGE
 		sharpness = sharpness_on
-		sharpness_flags = initial(sharpness_flags)
+		sharpness_flags = SHARP_TIP | SHARP_BLADE | INSULATED_EDGE | HOT_EDGE | CHOPWOOD
```

Comment:
Why are we writing it as initial anyway?  Isn't listing the flags way, way easier to read?